client/src/components/GameComponent.tsx

```diff
@@ -23,7 +23,7 @@ interface GameComponentProps {
 }
 export function GameComponent(props: GameComponentProps) {
   const { connection, token, sessionMetadata } = props;
-  if (connection != null || sessionMetadata.isGameEnd) {
+  if ((token != null && connection != null) || sessionMetadata.isGameEnd) {
```

Comment:
Won't this make it so only people authenticated with Google will be able to load the game? Since we removed the loginAnonymous logic, won't token be null for all players w/o google sign in? I mgiht be wrong

Reply:
you're totaly right, fixing